### PR TITLE
Add instructions for using Windows Nuget package

### PR DESCRIPTION
**Pull request by radhikaj**
In this PR
1. add instructions for installing prereqs and using Windows NuGet Package. Please note that most of this was lifted from https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/Contributors/WindowsManualInstallPrereqs.md 
2. Change c:\your\path\to\intel_and_dcap_nuget_packages to c:\oe_prereqs. This is because we have received feedback that the former is confusing and prevents a simple cut and paste work flow.

### CODE REVIEW

File: docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md [hunk 1]
```diff
@@ -10,11 +10,13 @@ C:\Intel SGX PSW for Windows v2.4.100.51291\PSW_EXE_RS2_and_before\Intel(R)_SGX_
 ## [Azure DCAP client for Windows](https://github.com/Microsoft/Azure-DCAP-Client/tree/master/src/Windows) [optional]
 
 Note that this is optional since you can choose an alternate implementation of the DCAP client or create your own.
-The Azure DCAP client for Windows is necessary if you would like to perform enclave attestation on a Azure Confidential Computing VM. It is available from [nuget.org](https://www.nuget.org/packages/Azure.DCAP.Windows/) and can be installed directly via:
+The Azure DCAP client for Windows is necessary if you would like to perform enclave attestation on a Azure Confidential Computing VM. It is available from [nuget.org](https://www.nuget.org/packages/Azure.DCAP.Windows/) and can be installed directly via command below.
+This example assumes that `C:\oe_prereqs` is where you would like the prerequisites to be installed.
```
Comment (johnkord) [inline docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md]: Nice

File: docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md [hunk 2]
```diff
@@ -10,11 +10,13 @@ C:\Intel SGX PSW for Windows v2.4.100.51291\PSW_EXE_RS2_and_before\Intel(R)_SGX_
...
 
 ```cmd
-nuget.exe install Azure.DCAP.Windows -ExcludeVersion -Version 0.0.2 -OutputDirectory C:\path\to\where\you\would\like\to\install\intel_and_dcap_nuget_packages
+nuget.exe install Azure.DCAP.Windows -ExcludeVersion -Version 0.0.2 -OutputDirectory C:\oe_prereqs
 ```
+This example assumes you would like to install the package to `C:\oe_prereqs`.
```
Comment (johnkord) [inline docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md]: Do we want to say this twice? :) Maybe it is useful...

File: docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md [hunk 3]
```diff
@@ -45,9 +47,9 @@ The following summary will assume that the contents were extracted to `C:\Intel
       ```
     - Note that `devcon.exe` is usually installed to `C:\Program Files (x86)\Windows Kits\10\tools\x64` which is not in the PATH environment variable by default.
 4. Install the DCAP nuget packages:
-    - The standalone `nuget.exe` [CLI tool](https://dist.nuget.org/win-x86-commandline/latest/nuget.exe) can be used to do this from the command prompt:
+    - The standalone `nuget.exe` [CLI tool](https://dist.nuget.org/win-x86-commandline/latest/nuget.exe) can be used to do this from the command prompt(assuming you would like to install the NuGet packages to `C:\oe_prereqs`):
```
Comment (johnkord) [inline docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md]: Insert space before the "(assuming"

File: docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md (hunk 2 above)
Comment (radhikaj) [reply docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md]: Got rid of the second one. Thanks

File: docs/GettingStartedDocs/install_oe_sdk-Windows.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
+# Install the Open Enclave SDK (On Windows Server 2016)
```
Comment (jxyang) [inline docs/GettingStartedDocs/install_oe_sdk-Windows.md]: Why is this file affected?

Comment (radhikaj) [reply docs/GettingStartedDocs/install_oe_sdk-Windows.md]: It is a new file.

File: docs/GettingStartedDocs/install_oe_sdk-Windows.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
+# Install the Open Enclave SDK NuGet Package
+
+## Platform requirements
+
+- A system with support for SGX1 with Flexible Launch Control (FLC).
+Note: To check if your system has support for SGX1 with or without FLC, please look [here](./SGXSupportLevel.md).
+
+- Windows Server 2016
```
Comment (jxyang) [inline docs/GettingStartedDocs/install_oe_sdk-Windows.md]: Could Windows 10 have FLC support?

File: docs/GettingStartedDocs/install_oe_sdk-Windows.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+Download the required Windows NuGet Package from [here](https://github.com/openenclave/openenclave/releases) and place it in a directory of your choice. Use the command below to install the NuGet package. In this example, we are placing the NuGet Package in `C:\openenclave_nuget` and installing it to `C:\oe`.
+
+```cmd
+ nuget.exe install open-enclave -Source C:\openenclave_nuget -OutputDirectory C:\oe -ExcludeVersion
+```
+
+Note: If it is an RC package, append `-pre` to the command above.
+
+## Verify the Open Enclave SDK install
```
Comment (jxyang) [inline docs/GettingStartedDocs/install_oe_sdk-Windows.md]: Install -> installation

File: samples/README_Windows.md [hunk 7]
```diff
@@ -47,7 +47,7 @@ set OpenEnclave_DIR=C:\openenclave\lib\openenclave\cmake
 ```cmd
 mkdir build
 cd build
-cmake .. -G Ninja -DNUGET_PACKAGE_PATH=C:\your\path\to\intel_and_dcap_nuget_packages
```
Comment (jxyang) [inline samples/README_Windows.md]: There is one more `your\path\to` in `building_oe_sdk.md`

File: docs/GettingStartedDocs/install_oe_sdk-Windows.md (hunk 5 above)
Comment (radhikaj) [reply docs/GettingStartedDocs/install_oe_sdk-Windows.md]: It should but is not a currently tested scenario. Will say we support it after we have tests.

File: samples/README_Windows.md (hunk 7 above)
Comment (radhikaj) [reply samples/README_Windows.md]: Thanks @jxyang. Fixed.

**Pull request merged**